yt_dlp/extractor/globo.py

```diff
@@ -38,126 +33,70 @@ class GloboIE(InfoExtractor):
-        'only_matching': True,
     }, {
         'url': 'globo:3607726',
         'only_matching': True,
-    }, {
-        'url': 'https://globoplay.globo.com/v/10248083/',
-        'info_dict': {
-            'id': '10248083',
-            'ext': 'mp4',
-            'title': 'Melhores momentos: Equador 1 x 1 Brasil pelas Eliminatórias da Copa do Mundo 2022',
-            'duration': 530.964,
-            'uploader': 'SporTV',
-            'uploader_id': '698',
-        },
-        'params': {
-            'skip_download': True,
-        },
     }]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
-        self._request_webpage(
-            HEADRequest('https://globo-ab.globo.com/v2/selected-alternatives?experiments=player-isolated-experiment-02&skipImpressions=true'),
-            video_id, 'Getting cookies')
-
+        video_view = '''
+        query getVideoView($videoId: ID!) {
+            video(id: $videoId) {
+                duration
+                description
+                headline
+                title {
+                    originProgramId
+                    headline
+                }
+            }
+        }
+        '''
         video = self._download_json(
-            f'http://api.globovideos.com/videos/{video_id}/playlist',
-            video_id)['videos'][0]
-        if not self.get_param('allow_unplayable_formats') and video.get('encrypted') is True:
-            self.report_drm(video_id)
-
-        title = video['title']
+            f'https://cloud-jarvis.globo.com/graphql?operationName=getVideoView&variables=%7B"videoId":"{video_id}"%7D&query={video_view}', video_id,
+            headers={'content-type': 'application/json', 'x-platform-id': 'web', 'x-device-id': 'desktop', 'x-client-version': '2024.12-5'})['data']['video']
+        title = video['headline']
+        uploader = video['title'].get('headline')
+        uploader_id = str_or_none(video['title'].get('originProgramId'))
 
         formats = []
         security = self._download_json(
-            'https://playback.video.globo.com/v2/video-session', video_id, f'Downloading security hash for {video_id}',
-            headers={'content-type': 'application/json'}, data=json.dumps({
+            'https://playback.video.globo.com/v4/video-session', video_id, f'Downloading resource info for {video_id}',
+            headers={'Content-Type': 'application/json'}, data=json.dumps({
                 'player_type': 'desktop',
                 'video_id': video_id,
                 'quality': 'max',
                 'content_protection': 'widevine',
-                'vsid': '581b986b-4c40-71f0-5a58-803e579d5fa2',
-                'tz': '-3.0:00',
+                'vsid': '2938bc7c-9376-d4b7-ee91-ce46dbbf9f4d',
+                'tz': '-03:00',
+                'version': 1,
             }).encode())
+        if not self.get_param('allow_unplayable_formats') and security['resource'].get('drm_protection_enabled') is True:
```

Comment:
we're not doing that unplayable param anymore
```suggestion
        if traverse_obj(video, ('resource', 'drm_protection_enabled', {bool})):
```